### ISSUE TITLE
Flag icons licensed CC-BY-SA-4.0, not compatible with BSD-3-Clause

### ISSUE THREAD

**Issue by andreasstieger** [mumble-voip/mumble#6411]
### Description

The Emojii One svg flag icons introduced in e94c8f7a03ba2fe31a178574be5eaf285f07a8e8, and updated from rectangle to circles in 5dbb124fedd04d983937b5c2c10caabb4943dfce are [licensed](https://github.com/mumble-voip/mumble/blob/e7f2c313670b957dd25ecbda858cfec49ffae8d4/icons/flags/LICENSE.md) under `CC-BY-SA-4.0`. 

These are then compiled into a Qt resource and hence the binary: 
https://github.com/mumble-voip/mumble/blob/e7f2c313670b957dd25ecbda858cfec49ffae8d4/src/mumble/CMakeLists.txt#L339

This creates a combination (adaptation) of a `CC-BY-SA-4.0` licensed work with `BSD-3-Clause`licensed code. They are not compatible: https://creativecommons.org/share-your-work/licensing-considerations/compatible-licenses/

Mumble should remove the Emojii One svg flag icons and replace them with compatible, permissively licensed data.

[flag-icons](https://github.com/lipis/flag-icons) are MIT licensed and could be used as a drop-in replacement while restoring flag dimensions. Alternatively... Unicode since 6.0 has [Regional indicator symbols](https://en.wikipedia.org/wiki/Regional_indicator_symbol) 

### Steps to reproduce

Build-time issue

### Mumble version

v1.5.629

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

Discovered by openSUSE license checker bot.

````
Found new license CC-BY-SA-2.1 (risk 3) not present in old report
CC-BY-SA-4.0: 1 files
mumble-1.5.629/icons/flags/LICENSE.processed.md
````

Development package: https://build.opensuse.org/package/show/games:tools/mumble

### Relevant log output

```shell
[   27s] [  0%] Generating mumble_flags.qrc
[   27s] cd /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble && /usr/bin/python3 /home/abuild/rpmbuild/BUILD/mumble-1.5.629/scripts/generate_flag_qrc.py --flag-dir /home/abuild/rpmbuild/BUILD/mumble-1.5.629/icons/flags --output /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/mumble_flags.qrc
[   27s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/mumble-1.5.629/build'
[...]
[   27s] [  1%] Automatic RCC for ../../mumble_flags.qrc
[   27s] cd /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble && /usr/bin/cmake -E cmake_autorcc /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble/CMakeFiles/mumble_client_object_lib_autogen.dir/AutoRcc_mumble_flags_IZWCZGADZF_Info.json RelWithDebInfo
[   27s] AutoRcc: Generating "SRC:/build/src/mumble/mumble_client_object_lib_autogen/IZWCZGADZF/qrc_mumble_flags.cpp", because it doesn't exist, from "SRC:/build/mumble_flags.qrc"
[   27s] /usr/lib64/qt5/bin/rcc -name mumble_flags -o /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble/mumble_client_object_lib_autogen/IZWCZGADZF/qrc_mumble_flags.cpp /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/mumble_flags.qrc
[...]
[  215s] [ 10%] Automatic RCC for ../../mumble_flags.qrc
[  215s] cd /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble && /usr/bin/cmake -E cmake_autorcc /home/abuild/rpmbuild/BUILD/mumble-1.5.629/build/src/mumble/CMakeFiles/mumble_client_object_lib_autogen.dir/AutoRcc_mumble_flags_IZWCZGADZF_Info.json RelWithDebInfo
[  215s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/mumble-1.5.629/build'
[  215s] [ 10%] Built target mumble_client_object_lib_arcc_mumble_flags
[...]
```

**Comment by davidebeatrici** [comment]
Thank you very much for bringing this to our attention!

My concern with the Unicode approach would be the potential rendering inconsistency between platforms and/or fonts.

`flag-icons` looks good to me.

**Comment by Hartmnt** [comment]
@Krzmbrzl Do I understand this correctly: To fix this we need to simply swap the SVG files for the flags, correct?

**Comment by Krzmbrzl** [comment]
That's my understanding as well, yes

**Comment by Hartmnt** [comment]
Ok, I guess we can do this for 1.5 as long as it does not require any code change. Do you prefer the 1:1 or the 3:4 aspect ratio of ``flag-icons``?

**Comment by andreasstieger** [comment]
> I guess this is something we'd want to consider to be done before and included in the 1.5 release.

Can you add it to [1.5 stable release](https://github.com/mumble-voip/mumble/projects/4) for tracking?

> Do you prefer the 1:1 or the 3:4 aspect ratio of flag-icons?

Personally I like the 4x3 visual language for flags. This is how this looks for me:

![Screenshot_2024-05-08_21-38-21](https://github.com/mumble-voip/mumble/assets/2847525/b0a00a94-f9e8-428e-a016-cc54e94aab2c)

**Comment by Hartmnt** [comment]
> Can you add it to [1.5 stable release](https://github.com/mumble-voip/mumble/projects/4) for tracking?

That thing looks outdated. The 1.5 stable release is scheduled to release this month and the features are basically frozen at this point :sweat_smile:
